### PR TITLE
Update SyntaxHighlight.tsx

### PR DESCRIPTION
**Pull request by utkarshraj779**
## Checklist

-   [ ] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This PR currently has none of the required changelog labels.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

File: src/components/views/elements/SyntaxHighlight.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ interface IProps {
 export default class SyntaxHighlight extends React.PureComponent<IProps> {
     public render(): JSX.Element {
         const { children: content, language } = this.props;
-        const highlighted = language ? hljs.highlight(content, { language }) : hljs.highlightAuto(content);
+        const highlighted = language ? hljs.highlight( { language },content) : hljs.highlightAuto(content);
```
Comment (weeman1337) [inline src/components/views/elements/SyntaxHighlight.tsx:29]: This doesn't look correct. Can you have another look?

Comment (utkarshraj779) [reply src/components/views/elements/SyntaxHighlight.tsx:29]: Ok I trie to make it better

Comment (utkarshraj779) [reply src/components/views/elements/SyntaxHighlight.tsx:29]: Okay I try to build this better

**Comment by weeman1337** [comment]
https://github.com/matrix-org/matrix-react-sdk/commit/24174826674caa5b6a2e5484ad27922e5c1adb63 already migrated to the new highlight js API. So there is nothing else to do here.

**Pull request closed without merging**